### ISSUE TITLE
Help text under input form

### ISSUE THREAD

**Issue by zufrieden** [swiss/styleguide#204]
![screenshot 2014-08-08 15 57 29](https://cloud.githubusercontent.com/assets/451236/3857495/f3aaf55e-1f03-11e4-90f6-da8eaa34cb3a.png)

Help text under the input field is not accessible via screen reeder. FInd the right format and implement it
